tokio/CHANGELOG.md

```diff
@@ -1,3 +1,45 @@
+# 1.30.0 (August 6, 2023)
+
```

Comment:
I think you need to mention here the MSRV bump, that's quite important.

Reply:
:+1: does that go under `### Changed` or should it be called out on its own at the top?

Reply:
IMO it should be put directly under `# 1.30.0 (Augest 6, 2023)` since it is quite important.